src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/IntegrationTestAuthOptions.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+using Microsoft.AspNetCore.Authentication;
+
+namespace Equinor.ProCoSys.IPO.WebApi.IntegrationTests
+{
+    internal class IntegrationTestAuthOptions : AuthenticationSchemeOptions
```

Comment:
Is this supposed to have anything else in it?

Reply:
no